public/js/app/components/statistics/MemosSummary.jsx

```diff
@@ -85,14 +100,16 @@ function MemosNumbersTable({ statisticsResult }) {
 }
 
 function MemosNumbersCharts({ statisticsResult }) {
+  console.log(statisticsResult)
   const chartNames = [
     'numberOfUniqWebAndPdfMemos',
     'numberOfMemosPublishedBeforeStart',
     'numberOfMemosPublishedBeforeDeadline',
   ]
   const { combinedMemosPerSchool: docsPerSchool, periods, year } = statisticsResult
-  const { schools = {} } = docsPerSchool
+  let { schools = {} } = docsPerSchool
   const [{ languageIndex }] = useWebContext()
+  schools = { ...schools, ALLS: addAllSchoolsData(docsPerSchool) }
```

Comment:
If you want to add only one property it is better to do schools.ALLS = add...

Reply:
done